tests/loop_tool/actions.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Please name test files with a `_test.py` suffix so that they are run by the pytest collector on `make install-test`